adafruit_ntp.py

```diff
@@ -55,6 +56,8 @@ def __init__(
             CircuitPython. CPython will determine timezone automatically and adjust (so don't use
             this.) For example, Pacific daylight savings time is -7.
         :param int socket_timeout: UDP socket timeout, in seconds.
+        :param int cache_seconds: how many seconds to use a cached result from NTP server
```

Comment:
I agree that's a good default but I'd rather not change behavior. Instead, I'd default to None to indicate following the server.

Reply:
I'll default it to zero- min(int, None) is an error.

Reply:
okay, pushed the 'zero', and then changed all of the examples to use the 3600 value, since most people just copy/paste example code anyhow. Did it in a standalone commit in case you disagree and want me to revert.